### PR TITLE
test: 12 tests from others

### PR DESCRIPTION
**Pull request by mintbomb27**
## Related Issue(s)
#546 

## Description
Contains 12 kuttl tests from others category

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [ ] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

File: other/require-imagepullsecrets/pod-good.yaml
Comment (chipzoller) [inline other/require-imagepullsecrets/pod-good.yaml:1]: Please incorporate quay.io into at least one good Pod.

File: other/require-netpol/netpol.yaml
Comment (chipzoller) [inline other/require-netpol/netpol.yaml:1]: I'd like to see one possibly two more NetworkPolicies as part of this test in which the labels map has more than just one entry. We want to stress test the `label_match()` filter well and I think in previous tests I've seen those tests have been fairly basic. Let's bolster it here.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Nice. Couple of requests.

File: other/require-netpol/netpol.yaml
Comment (mintbomb27) [reply other/require-netpol/netpol.yaml:1]: Roger that.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**